### PR TITLE
feat: add 1.0 blog post

### PR DESCRIPTION
**Pull request by toddbaert**
Add blog post about 1.0 and spec stabilization.

Direct preview link: https://deploy-preview-160--lucky-creponne-baf9b3.netlify.app/blog

### CODE REVIEW

File: blog/21-10-2022-hardening-and-1-0-sdks.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+date: 2022-10-21
+authors: [toddbaert]
+description: "OpenFeature is ready for primetime: stabilizing the specification and 1.0 SDK releases"
+keywords: [release, "1.0", sdk, openFeature, java, go, ".net", dotnet, js, javascript, specification, spec]
+draft: false
+---
+
+Early this year, OpenFeature announced its intent to bring a standard to the rapidly growing development practice of feature flagging.
+In June it was accepted as a Cloud Native Computing Foundation Sandbox project.
+Now, we're pleased to announce a new milestone: OpenFeature has released 1.0 versions of it's [.NET](https://github.com/open-feature/dotnet-sdk), [Go](https://github.com/open-feature/go-sdk), [Java](https://github.com/open-feature/java-sdk), and [JavaScript](https://github.com/open-feature/js-sdk) SDKs!
```
Comment (tcarrio) [inline blog/21-10-2022-hardening-and-1-0-sdks.md:13]: s/it's/its/

File: blog/21-10-2022-hardening-and-1-0-sdks.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+In addition to those mentioned above, experimental features available in the 1.0 SDKs include:
+
+- Hooks, for adding arbitrary behavior to feature flag evaluation, ideal for telemetry integration, validation, and logging.
+- The Evaluation Context interfaces, used as the basis for dynamic flag evaluation.
+
+## What's next?
+
+Our goal in the upcoming months will be to harden our existing experimental features. Additionally, we'll work to develop and standardize new capabilities, including: client-side feature flagging, improved cloud native tooling, and implicit transaction-scoped data propagation of contextual attributes.
+Furthermore, we're working on SDKs for additional languages, including PHP, Python, and Ruby.
```
Comment (tcarrio) [inline blog/21-10-2022-hardening-and-1-0-sdks.md:33]: Perhaps link to these SDK repositories

**Review by tcarrio — APPROVED** [review]
Grammar and possible linking opportunities

**Comment by tcarrio** [comment]
Additionally, this is a preemptive post, correct? In anticipation of the 1.0 spec release? Asking since I still don't see a 1.0 in the spec repo :slightly_smiling_face:

If `0.5.0` is essentially `1.0` that would be good to know as well

File: blog/21-10-2022-hardening-and-1-0-sdks.md (hunk 2 above)
Comment (tcarrio) [inline blog/21-10-2022-hardening-and-1-0-sdks.md:33]:
```suggestion
Furthermore, we're working on SDKs for additional languages, including [PHP](https://github.com/open-feature/php-sdk), [Python](https://github.com/open-feature/python-sdk), and [Ruby](https://github.com/open-feature/ruby-sdk).
```

File: blog/21-10-2022-hardening-and-1-0-sdks.md (hunk 1 above)
Comment (toddbaert) [reply blog/21-10-2022-hardening-and-1-0-sdks.md:13]: Nice catch, thanks!

**Comment by toddbaert** [comment]
> Additionally, this is a preemptive PR, correct? In anticipation of the 1.0 spec release? Asking since I still don't see a 1.0 in the spec repo slightly_smiling_face
> 
> If `0.5.0` is essentially `1.0` that would be good to know as well

We are not doing a 1.0 release of the spec, but have added `hardening` labels to certain subsections. Only the mentioned SDKs are 1.0 (2/4 thus far, but hopefully it's 4/4 soon). Should we add clarification on that?

File: blog/21-10-2022-hardening-and-1-0-sdks.md (hunk 2 above)
Comment (toddbaert) [reply blog/21-10-2022-hardening-and-1-0-sdks.md]: Looks like you did this.

**Comment by justinabrahms** [comment]
I want to find a 3rd bullet point for the experimental sdk features. Also, "hardening" sounds almost stable but not actually stable. Can it regress? If not, we should find a better name. If so, we should mention it. :)

**Comment by toddbaert** [comment]
> I want to find a 3rd bullet point for the experimental sdk features.

I'm sure I can think of one.

> Also, "hardening" sounds almost stable but not actually stable. Can it regress? If not, we should find a better name. If so, we should mention it. :)

The "hardening" language was agreed on here: https://github.com/open-feature/spec/pull/139. It can't be regressed; we agreed we wanted a unidirectional graduation.

I'm open to changing the term if the meaning doesn't change. Any proposals?

**Comment by justinabrahms** [comment]
This works for me. Re read that post and it's about which version number changes on update, which works for me.

**Comment by toddbaert** [comment]
@justinabrahms 

I've added one more point to the experimental features. Right now, we have 4 sections in the SDK, 2 of which are `hardening` and 2 of which are sill `experimental`. The point I've added is "experimental context propagation" (passing context attrs on the async continuation or thread-local storage, etc) which is currently only implemented experimentally in the JS-SDK.

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**